### PR TITLE
add support for eu-north-1 region

### LINKED ISSUE #6818: Add support for region eu-north-1
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The new Stockholm region is released, and has the identifier `eu-north-1`

```
        {
            "Endpoint": "ec2.eu-north-1.amazonaws.com",
            "RegionName": "eu-north-1"
        },
```

We should add support for this new region, however, I don't know enough about the aws provider to do so myself.

### References
Currently this hasn't been posted on either the AWS blog, but the region shows up in the aws-cli when running `aws ec2 describe-regions` as well as on the [AWS Regions and Endpoints](https://docs.aws.amazon.com/general/latest/gr/rande.html) page.

### PR DESCRIPTION
**Pull request by sunilkumarmohanty**
Fixes #6818

### CODE REVIEW

**Comment by ewbankkit** [comment]
Thanks. I think this also fixes https://github.com/terraform-providers/terraform-provider-aws/issues/6832.

**Comment by ewbankkit** [comment]
@bflad Do we want to use the constants defined in the AWS SDK [`endpoints`](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/#pkg-constants) package instead of the string literals?
We could then have a test case for each region?

File: aws/hosted_zones.go [hunk 1]
```diff
@@ -6,22 +6,24 @@ import "fmt"
...
 	"ca-central-1":   "Z1QDHH18159H29",
+	"eu-central-1":   "Z21DNDUVLTQW6Q",
+	"eu-north-1":     "Z3BAZG2TWCNX0D",
+	"eu-west-1":      "Z1BKCTXD74EZPE",
+	"eu-west-2":      "Z3GKZC51ZF0DB4",
+	"eu-west-3":      "Z3R1K369G5AVDG",
 	"sa-east-1":      "Z7KQH4QJS55SO",
+	"us-east-1":      "Z3AQBSTGFYJSTF",
+	"us-east-2":      "Z2O1EMRO9K5GLX",
 	"us-gov-west-1":  "Z31GFT0UA1I2HV",
```
Comment (bflad) [inline aws/hosted_zones.go]: While we're making adjustments to this pull request, we can add `us-gov-east-1`, which has the same Route53 Hosted Zone ID as `us-gov-west-1` according to the [GovCloud (US-East) Endpoints documentation](https://docs.aws.amazon.com/govcloud-us/latest/ug-east/using-govcloud-endpoints.html)

```suggestion
	"us-gov-east-1":  "Z31GFT0UA1I2HV",
	"us-gov-west-1":  "Z31GFT0UA1I2HV",
```

**Comment by bflad** [comment]
> @bflad Do we want to use the constants defined in the AWS SDK endpoints package instead of the string literals?

I'm ambivalent either which way -- we may add information for AWS regions outside AWS Commercial, GovCloud (US), and China, which would not be present as constants in that file.

File: aws/data_source_aws_elb_service_account.go [hunk 2]
```diff
@@ -11,19 +11,22 @@ import (
 var elbAccountIdPerRegionMap = map[string]string{
 	"ap-northeast-1": "582318560864",
 	"ap-northeast-2": "600734575887",
+	"ap-northeast-3": "765225791966",
```
[the remark this answers is not in the document]
Comment (sunilkumarmohanty) [reply aws/data_source_aws_elb_service_account.go]: Some stupid copy paste mistake

**Comment by bflad** [comment]
This has been released in [version 1.52.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1520-december-13-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Pull request merged**